docs/user_manual/calculations.md

```diff
@@ -8,25 +8,83 @@ SPDX-License-Identifier: MPL-2.0
 
 ## Calculation types
 With power-grid-model it is possible to perform two different types of calculation:
-- [Power flow](###Power flow algorithms): a "what-if" scenario calculation. This calculation can be performed by using the {py:class}`calculate_power_flow <power_grid_model.PowerGridModel.calculate_power_flow>` method. An example of usage of the power-flow calculation function is given in [Power flow Example](ex_power_flow)
-- [State estimation](###State estimation algorithms): a statistical method that calculates the most probabilistic state of the grid, given sensor values with an uncertainty. This calculation can be performed by using the {py:class}`calculate_state_estimation <power_grid_model.PowerGridModel.calculate_state_estimation>` method. An example of usage of the power-flow calculation function is given in [State Estimation Example](ex_state_est)
+- [Power flow](#power-flow-algorithms): a "what-if" scenario calculation. This calculation can be performed by using the {py:class}`calculate_power_flow <power_grid_model.PowerGridModel.calculate_power_flow>` method. An example of usage of the power-flow calculation function is given in [Power flow Example](../examples/Power%20Flow%20Example.ipynb)
+- [State estimation](#state-estimation-algorithms): a statistical method that calculates the most probabilistic state of the grid, given sensor values with an uncertainty. This calculation can be performed by using the {py:class}`calculate_state_estimation <power_grid_model.PowerGridModel.calculate_state_estimation>` method. An example of usage of the power-flow calculation function is given in [State Estimation Example](../examples/State%20Estimation%20Example.ipynb)
 
 ### Calculation types explained
-TODO: 
-- What is the difference between power flow and state estimation
-- When should you use which? Maybe small example (physical, not code)
+
+#### Power flow
+Power flow is a "what-if" based grid calculation that will calculate the node voltages and the power flow through the branches, based on assumed load/generation profiles.
+Some typical use-cases are network planning and contingency analysis.
+
+Input:
+- Network data: topology + component attributes
+- Assumed load/generation profile
+
+Output:
+- Node voltage magnitude and angle
+- Power flow through branches
+
+#### State estimation
+State estimation is a statistical calculation method that determines the most probable state of the grid, based on
+network data and measurements. Measurements meaning power flow or voltage values with some kind of uncertainty, which were 
+either measured, estimated or forecasted.
+
+Input:
+- Network data: topology + component attributes
+- Power flow / voltage measurements with uncertainty
```

Comment:
I think it would be good to emphasize that _measurements_ don't have to be real measurements, but can be estimates or predictions as well. And maybe we can also emphasize at _uncertainty_ is not just being uncertain, but you have to specify _how_ uncertain your measurements are.

I now that this is all very logical, if you are familiar with the literature, like Phase to Phase, but a many users are not.